### ISSUE TITLE
Tech support question only, sorry.

### ISSUE THREAD

**Issue by jaxjexjox** [pymedusa/Medusa#10826]
Does "Ignored words" honour quotations?
So it will only exclude the "two words" in a combined form within "quotation marks"?

Does that make sense?   I'd like to avoid very specific groups of words.
Thank you, sorry for the poor request.

**Comment by p0psicles** [comment]
Yes it will ony exclude the combined words. But you don't need to use quotation marks. The words or combination of words are send separate to the backend.

**Comment by jaxjexjox** [comment]
I absolutely need it to only recognise quotation combined words.
My wife watches 90 Day fiance (I know,.. I know..........)

I would like to set my exclusions, for the root show (simply titled "90 Day Fiance") to exclude the 'secondary' sub 90 Day products which they shovel out.

Example
90 Day Fiance  The Other Way
90 Day Fiance  Happily Ever After
90 Day Fiance  Before the 90 Days


If I add an ignore rule for "the other way"
and it actually ignores
the OR other OR way

Then a genuine episode of the root show, may be missed if it contains those 3 words in the episode title.
Example
90 Day Fiance S05E02  _Other_ Stuff going on.mkv
or
90 Day Fiance S07E03 _The_ other guys.mkv

Does this make sense?
Therefore, I'd like my ignore words to recognise outright blocks in quotation marks, ensuring it doesn't pull in 'sub shows'

**Comment by BenjV** [comment]
To my understanding it is "Ignored Words" and not "Ignored Sentence" or "Ignored String".
So if you put a sentence in that field it will be treated as multiple words.

**Comment by strike84** [comment]
Ignored words is not the right solution in this case.  There are probably scene exceptions set for that show. Edit the show and select advanced. If I'm right those shows you want exuded are probably listed under scene exceptions. Just remove the ones you don't want.  If nothing is there just put the name you want medusa to search for. Then medusa will only search for that show.

**Comment by jaxjexjox** [comment]
> Ignored words is not the right solution in this case. There are probably scene exceptions set for that show. Edit the show and select advanced. If I'm right those shows you want exuded are probably listed under scene exceptions. Just remove the ones you don't want. If nothing is there just put the name you want medusa to search for. Then medusa will only search for that show.

That's the problem though.

The show is called 90 Day Fiance.
They also make other products:
90 Day Fiance The Other Way
90 Day Fiance Happily Ever After
90 Day Fiance Before the 90 Days

All starting with 90 Day Fiance in the title.

I already have 90 Day Fiance AS a scene exception, to get past the issue of "é"  (90 Day Fiancé)
So is Medusa smart enough to search for /only/ 90 Day Fiance, without any other text in the title, before the episode numbering (S0xE0x) etc

**Comment by strike84** [comment]
> So is Medusa smart enough to search for /only/ 90 Day Fiance, without any other text in the title, before the episode numbering (S0xE0x) etc

It should be as long as you only have 90 Day Fiance as a scene exception.

**Comment by p0psicles** [comment]
There is a tool in config - postprocessing - (guessit) tab. There you can test how different release names match against your shows
